HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -743,7 +743,7 @@ def self.run_validator(hpxml_doc)
       },
 
       ## [LightingGroup]
-      'LightingGroup[LightingType[LightEmittingDiode | CompactFluorescent | FluorescentTube] and Location[text()="interior" or text()="exterior" or text()="garage"]]' => {
+      '/HPXML/Building/BuildingDetails/Lighting/LightingGroup[LightingType[LightEmittingDiode | CompactFluorescent | FluorescentTube] and Location[text()="interior" or text()="exterior" or text()="garage"]]' => {
```

Comment:
Good catch! Can you issue a separate PR with just this bugfix? I'd like to get this fix into master sooner than waiting on the full schematron capability.